apollo-router/src/plugins/cache/entity.rs

```diff
@@ -389,12 +431,16 @@ impl InnerCacheService {
                         self.storage,
                         self.subgraph_ttl,
                         &mut response,
-                        cache_control,
+                        cache_control.clone(),
                         cache_result.0,
                         is_known_private,
                         private_id,
                     )
                     .await?;
+
+                    // FIXME: will the header be duplicated?
```

Comment:
Does it need to be fixed as part of this PR ?

Reply:
I just checked, the `HeaderMap::insert` method replaces all existing entries, so it is fine